src/components/tag/Tag.props.ts

```diff
@@ -0,0 +1,13 @@
+import { TTextSizeToken } from '../text/Text.types';
+
+import { TTagType } from './Tag.types';
+
+type PTag = {
+    type: TTagType;
```

Comment:
`type` is probably not the best property name, since it exists on some HTML elements as well (with different values) and when passed down (it is in the `DEFAULT_PROPERTY_WHITELIST`) it could lead to unwanted behaviour - not to speak of semantics and a11y.

Maybe call it `variant`? We use this term a lot on other components